pkg/gateway/operations/deleteobjects.go

```diff
@@ -27,6 +27,13 @@ func (controller *DeleteObjects) RequiredPermissions(_ *http.Request, _ string)
 }
 
 func (controller *DeleteObjects) Handle(w http.ResponseWriter, req *http.Request, o *RepoOperation) {
+	// verify we only handle delete request
+	query := req.URL.Query()
+	if !query.Has("delete") {
+		_ = o.EncodeError(w, req, nil, gerrors.ERRLakeFSNotSupported.ToAPIErr())
+		return
+	}
```

Comment:
Can we `HandleUnsupported` here?  One day we'll add delete-part or something, and even before that this is less uniform so I need to think harder when reading it.

Reply:
Pleased?  (Or say no, that also works -- prefer not to leave this open)